tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -63,6 +90,13 @@ class ExplainType:
     @property
     def value(self) -> int: ...
 
+class DuckDBPyType:
```

Comment:
I know we've done it with the other classes, but I don't see why we need to prefix them all with `DuckDB`? Python has perfectly good namespacing as it is, and you can rename on import too

Reply:
I mostly did it to be consistent with the other classes, I have no strong preference either way.
I think internally since they live in the `duckdb` namespace having the DuckDBPy prefix is preferable, but that doesn't mean they have to be exposed as such to the python world

Reply:
I think I'll leave it as-is for now